financialaid/serializers.py

```diff
@@ -109,7 +109,7 @@ def validate(self, data):
                 self.instance.status != FinancialAidStatus.PENDING_MANUAL_APPROVAL):
             raise ValidationError("Cannot approve application that is not pending manual approval.")
         if (data['action'] == FinancialAidStatus.PENDING_MANUAL_APPROVAL and
-                self.instance.status != FinancialAidStatus.PENDING_DOCS):
+                self.instance.status not in [FinancialAidStatus.PENDING_DOCS, FinancialAidStatus.DOCS_SENT]):
             raise ValidationError("Cannot mark documents as received for application not pending docs.")
```

Comment:
it looks like this validation message needs to change, and tbh the language is kind of obtuse as it is right now